http-poster/src/main/java/com/expedia/www/haystack/pipes/httpPoster/HttpPostAction.java

```diff
@@ -80,15 +81,15 @@
         this.invalidProtocolBufferExceptionLogger = invalidProtocolBufferExceptionLogger;
 
         String msg = String.format(STARTUP_MESSAGE,
-                this.httpPostConfigurationProvider.url(), this.httpPostConfigurationProvider.pollpercent());
+                this.httpPostConfigurationProvider.getUrl(), this.httpPostConfigurationProvider.getPollPercent());
         this.httpPostActionLogger.info(msg);
     }
 
     @Override
     public void apply(String key, Span span) {
         timersAndCounters.incrementRequestCounter();
         timersAndCounters.recordSpanArrivalDelta(span);
-        if(random.nextInt(ONE_HUNDRED_PERCENT) < Integer.parseInt(httpPostConfigurationProvider.pollpercent())) {
+        if(random.nextInt(ONE_HUNDRED_PERCENT) < Integer.parseInt(httpPostConfigurationProvider.getPollPercent())) {
```

Comment:
Please do a search in all code for if( and change to if ( to keep consistency :).

Reply:
👍